docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1135,6 +1135,51 @@ In this example, if the `remove` processor does fail, then
 the array elements that have been processed thus far will
 be updated.
 
+
+Another example, using the attachment processor:
```

Comment:
I'd also may be add a new section in attachment processor doc (`docs/plugins/ingest-attachment.asciidoc`):

```
[[ingest-attachment-with-arrays]]
==== Using the Attachment Processor with arrays

```